### ISSUE TITLE
pictures of the data set

### ISSUE THREAD

**Issue by sukeey** [Supermaxman/MMVax-Stance#1]
Hello author, I am very interested in your paper and data set. I would like to ask you how to obtain pictures of the data set. Thank you again for your work.

**Comment by Supermaxman** [comment]
Hello @sukeey! Unfortunately due to Twitter / X's TOS and our IRB we cannot release the raw social media posts, including images.  However, if you have IRB approval and a Twitter / X developer account you can download the raw posts (and images) yourself using the tweet IDs. This process is commonly referred to as "Hydrating" a tweet.

**Comment by sukeey** [comment]
So please ask, does the text content of the data also need to be crawled based on the ID? I did not find the data of the text content.Sorry to trouble you, I really want to use this data set, but I really can't start at the moment.

**Comment by Supermaxman** [comment]
Yes, the tweet ID should allow you to collect both the textual content and the image of a post.

**Comment by sukeey** [comment]
Thank you very much for your reply, and I would like to ask you a question. What is the difference between the last two columns of "total" and "tweets" in Table 3 of your paper? I just did a master's degree and I didn't understand it. The ID I found in the annotations folder of your project is "tweets". Will I use the tweets column when using your data set, with a total of 6918 pieces of data?Thank you very much and looking forward to your reply。
![](https://img.x3.work/img/lsq/2024/01/03/100751.jpg)
